### PR TITLE
add bundler audit

### PR DESCRIPTION
**Pull request by GrrrD**

### CODE REVIEW

**Comment by mykecameron** [comment]
niiiice

:octocat:

File: spec/bundler_audit_spec.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+require 'bundler/audit/scanner'
+
+describe 'bundle' do
+  it 'is secure' do
+    vulnerabilities = Bundler::Audit::Scanner.new.scan.to_a
+    # if failing, run `bundle-audit update` at the terminal update ruby-advisory-db
```
Comment (mattdoller) [inline spec/bundler_audit_spec.rb]: thoughts on putting this comment into the error message of the assertion itself, so it's automatically dumped to the console whenever a dev is running the spec?

something like:

```ruby
expect(vulnerabilities).to be_empty(<<-MESSAGE)
  Vulnerabilities were discovered - run the following commands...
MESSAGE
```

**Comment by mattdoller** [comment]
👍 one comment, but not blocking

File: spec/bundler_audit_spec.rb (hunk 1 above)
Comment (GrrrD) [inline spec/bundler_audit_spec.rb]: good idea

**Comment by mattdoller** [comment]
🚢

**Pull request merged**